test/services/audit/AuditServiceSpec.scala

```diff
@@ -16,70 +16,234 @@
+        actualExtendedDataEvent.tags mustBe expectedExtendedDataEvent.tags
+        actualExtendedDataEvent.detail mustBe expectedExtendedDataEvent.detail
+      }
     }
 
-    "audit a movement" in {
-      val dataToAudit = Map(
-        EventData.movementReference.toString -> "",
-        EventData.eori.toString -> "GB12345678",
-        EventData.messageCode.toString -> "EAL",
-        EventData.ucr.toString -> "UCR",
-        EventData.ucrType.toString -> "D",
-        EventData.submissionResult.toString -> "200"
-      )
-      val data =
-        MovementRequest(
-          eori = "GB12345678",
-          choice = MovementType.Arrival,
-          consignmentReference = ConsignmentReferences("UCR", "D"),
-          movementDetails = MovementDetailsRequest("dateTime")
+    "call AuditConnector sendEvent method with correct data when auditing submission result" when {
+
+      "used for Shut a Mucr" in {
+
+        val auditTags = auditTagsResult("ShutMucr")
+        val auditDetail =
+          Map(EventData.eori.toString -> CommonTestData.validEori, EventData.mucr.toString -> "mucr", EventData.submissionResult.toString -> "200")
+        val expectedDataEvent =
+          DataEvent(auditSource = "appName", auditType = AuditType.AuditShutMucr.toString, tags = auditTags, detail = auditDetail)
+
+        service.auditShutMucr(CommonTestData.validEori, "mucr", "200")
+
+        val actualDataEvent = sentDataEvent
+        actualDataEvent.auditSource mustBe expectedDataEvent.auditSource
+        actualDataEvent.auditType mustBe expectedDataEvent.auditType
+        actualDataEvent.tags mustBe expectedDataEvent.tags
+        actualDataEvent.detail mustBe expectedDataEvent.detail
+      }
+
+      "used for Association" in {
+
+        val auditTags = auditTagsResult("Associate")
+        val auditDetail =
+          Map(
+            EventData.eori.toString -> CommonTestData.validEori,
+            EventData.mucr.toString -> "mucr",
+            EventData.ducr.toString -> "ducr",
+            EventData.submissionResult.toString -> "200"
+          )
+        val expectedDataEvent =
+          DataEvent(auditSource = "appName", auditType = AuditType.AuditAssociate.toString, tags = auditTags, detail = auditDetail)
+
+        service.auditAssociate(CommonTestData.validEori, "mucr", "ducr", "200")
+
+        val actualDataEvent = sentDataEvent
+        actualDataEvent.auditSource mustBe expectedDataEvent.auditSource
+        actualDataEvent.auditType mustBe expectedDataEvent.auditType
+        actualDataEvent.tags mustBe expectedDataEvent.tags
+        actualDataEvent.detail mustBe expectedDataEvent.detail
+      }
+
+      "used for Disassociation" in {
+
+        val auditTags = auditTagsResult("Disassociate")
+        val auditDetail =
+          Map(EventData.eori.toString -> CommonTestData.validEori, EventData.ducr.toString -> "ducr", EventData.submissionResult.toString -> "200")
```

Comment:
Shouldnt it be
```
Map(EventData.eori.toString -> CommonTestData.validEori, EventData.ucr.toString -> "ducr", EventData.submissionResult.toString -> "200")
``` 

`EventData.ucr.toString` being the key bit

Reply:
It seems that it was not addressed in this service. Change is comming in the next push.